dts/bindings/sensor/maxim,max1726x.yaml

```diff
@@ -45,3 +45,46 @@ properties:
       type: int
       required: true
       description: Charge voltage in mV
+
+    hibernate-threshold:
+      type: int
+      required: false
+      default: 7
```

Comment:
I think the rule is that you need to explain in description why this is the defaut (e.g. hardware defaults to 7 at startup). Fix globally.

Reply:
Done, thanks for your feedback !

Reply:
> Done, thanks for your feedback !

This needs to be done for all the defaults. Additionally some properties say there is a default but none is present in the binding. That looks inconsistent; what's going on?